src/sentry/workflow_engine/handlers/condition/event_frequency_base_handler.py

```diff
@@ -139,7 +140,12 @@ def get_value_from_groups(
 
     @abstractmethod
     def batch_query(
-        self, group_ids: set[int], start: datetime, end: datetime, environment_id: int | None
+        self,
+        group_ids: set[int],
+        start: datetime,
+        end: datetime,
+        environment_id: int | None,
+        interval: str,
```

Comment:
needed to add `interval` because it's used in the percent sessions handler. it's always available on the `DataCondition` so it's not a problem to pass it. we use it to calculate the `start` & `end` for other slow condition handlers